cli/src/runner.rs

```diff
@@ -157,12 +176,16 @@ pub async fn extract_failed_tests(
                     if failure {
                         let name = case.original_name;
                         log::debug!("Test failed: {} -> {}", parent_name, name);
-                        failures.push(Test {
+                        let mut test = Test {
                             parent_name: parent_name.clone(),
```

Comment:
Why do we need to clone these fields if you are already moving cases out of `suite.cases` ?